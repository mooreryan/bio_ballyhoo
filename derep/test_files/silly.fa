>s1 duplicated
ACTG
>s2 duplicated
AC
TG
>s3 duplicated
a
c
t
g
