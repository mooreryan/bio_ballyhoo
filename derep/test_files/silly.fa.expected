>s3 duplicated
ACTG
